modules/nf-core/sentieon/haplotyper/main.nf

```diff
@@ -3,19 +3,17 @@ process SENTIEON_HAPLOTYPER {
     label 'process_medium'
     label 'sentieon'
 
-    secret 'SENTIEON_LICENSE_BASE64'
-
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/sentieon:202308.02--h43eeafb_0' :
-        'biocontainers/sentieon:202308.02--h43eeafb_0' }"
+        'oras://community.wave.seqera.io/library/sentieon:202308.02--ffce1b7074ce9924' :
+        'nf-core/sentieon:202308.02--c641bc397cbf79d5' }"
 
     input:
-    tuple val(meta), path(input), path(input_index), path(intervals)
-    path  fasta
-    path  fai
-    path  dbsnp
-    path  dbsnp_tbi
+    tuple val(meta), path(input), path(input_index), path(intervals), path(recal_table)
+    tuple val(meta1), path(fasta)
+    tuple val(meta2), path(fai)
+    tuple val(meta3), path(dbsnp)
+    tuple val(meta4), path(dbsnp_tbi)
```

Comment:
Did you test this? I am surprised this didn't require changes to the module call in `bam_variant_calling_germline`

Reply:
NOT AT ALL, I thought we fixed sentieon server so we could do CI

Reply:
:facepalm: 

Anyhow, I can see that all sentieon modules are using `oras`, so we still need to update those one ore time